### PR TITLE
Cleaned up urls.py. Fixed #1472.

### LINKED ISSUE #1472: Use url() in urls.py

### PR DESCRIPTION
**Pull request by normanjaeckel**

### CODE REVIEW

File: openslides/urls.py [hunk 1]
```diff
@@ -40,9 +35,13 @@
...
 
 # TODO: move this patterns into core or the participant app
+js_info_dict = {'packages': []}
+
 urlpatterns += patterns(
     '',
-    (r'^jsi18n/$', 'django.views.i18n.javascript_catalog', js_info_dict),
+    url(r'^jsi18n/$',
+        'django.views.i18n.javascript_catalog',
+        js_info_dict),
```
Comment (ostcar) [inline openslides/urls.py]: The js_info_dict can be removed right away. We definitely need another way to handle translation in angular

**Comment by ostcar** [comment]
One comment. Please fix it or merge directly

File: openslides/urls.py (hunk 1 above)
Comment (normanjaeckel) [inline openslides/urls.py]: Done.

**Comment by ostcar** [comment]
ok

**Pull request merged**